app/controllers/items_controller.rb

```diff
@@ -2,10 +2,12 @@ class ItemsController < ApplicationController
   def index
 
   end
+
   def new
     @item = Item.new
 
   end
+
   def create
       @item = Item.new(item_params)
```

Comment:
Still indentation is not correct. Please fix all indentation for all the files using rubocop